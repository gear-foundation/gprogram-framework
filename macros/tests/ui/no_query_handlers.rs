use gprogram_framework_macros::{gprogram, query_handlers};

#[gprogram]
mod gprogram {
    use super::*;

    query_handlers!();
}

fn main() {}
