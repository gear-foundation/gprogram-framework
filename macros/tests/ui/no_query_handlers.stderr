error: No handlers found. Please either define one or remove the macro usage
 --> tests/ui/no_query_handlers.rs:7:5
  |
7 |     query_handlers!();
  |     ^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `query_handlers` (in Nightly builds, run with -Z macro-backtrace for more info)
